csharp/ql/lib/semmle/code/csharp/Type.qll

```diff
@@ -204,6 +204,8 @@ class ValueOrRefType extends Type, Attributable, @value_or_ref_type {
     this.hasMethod(c)
     or
     this.hasMember(c.(Accessor).getDeclaration())
+    or
+    c = this.getAConstructor()
```

Comment:
Alternatively replace `hasMethod` in the first line of the predicate with `hasMember` - I think that's equivalent.